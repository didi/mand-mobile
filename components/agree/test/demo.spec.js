import Demo0 from './cases/demo0'
import Demo1 from './cases/demo1'
import Demo2 from './cases/demo2'
import Demo3 from './cases/demo3'
import {renderToString} from '@vue/server-test-utils'

describe('Agree', () => {
  test(`Selected`, () => {
    expect(renderToString(Demo0)).toMatchSnapshot()
  })
  test(`Unselected`, () => {
    expect(renderToString(Demo1)).toMatchSnapshot()
  })
  test(`Disabled selected`, () => {
    expect(renderToString(Demo2)).toMatchSnapshot()
  })
  test(`Disabled unselected`, () => {
    expect(renderToString(Demo3)).toMatchSnapshot()
  })
})
