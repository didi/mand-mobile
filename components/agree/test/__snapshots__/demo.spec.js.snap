// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Agree Disabled selected 1`] = `
<div class="md-example-child md-example-child-agree md-example-child-2">
  <div class="md-agree disabled">
    <div class="md-agree-icon checked">
      <div class="md-agree-icon-container"><i class="md-icon icon-font md-icon-checked md" style="color:;"></i> <i class="md-icon icon-font md-icon-check md" style="color:;"></i></div>
    </div>
    <div class="md-agree-content">
      本人承诺投保人已充分了解本保险产品，并保证投保信息的真实性，理解并同意
    </div>
  </div>
</div>
`;

exports[`Agree Disabled unselected 1`] = `
<div class="md-example-child md-example-child-agree md-example-child-3">
  <div class="md-agree disabled">
    <div class="md-agree-icon">
      <div class="md-agree-icon-container"><i class="md-icon icon-font md-icon-checked md" style="color:;"></i> <i class="md-icon icon-font md-icon-check md" style="color:;"></i></div>
    </div>
    <div class="md-agree-content">
      本人承诺投保人已充分了解本保险产品，并保证投保信息的真实性，理解并同意
    </div>
  </div>
</div>
`;

exports[`Agree Selected 1`] = `
<div class="md-example-child md-example-child-agree md-example-child-0">
  <div class="md-agree">
    <div class="md-agree-icon checked">
      <div class="md-agree-icon-container"><i class="md-icon icon-font md-icon-checked md" style="color:;"></i> <i class="md-icon icon-font md-icon-check md" style="color:;"></i></div>
    </div>
    <div class="md-agree-content">
      本人承诺投保人已充分了解本保险产品，并保证投保信息的真实性，理解并同意<a>《投保须知》</a>, <a>《保险条款》</a></div>
  </div>
</div>
`;

exports[`Agree Unselected 1`] = `
<div class="md-example-child md-example-child-agree md-example-child-1">
  <div class="md-agree">
    <div class="md-agree-icon">
      <div class="md-agree-icon-container"><i class="md-icon icon-font md-icon-checked md" style="color:;"></i> <i class="md-icon icon-font md-icon-check md" style="color:;"></i></div>
    </div>
    <div class="md-agree-content">
      本人承诺投保人已充分了解本保险产品，并保证投保信息的真实性，理解并同意
    </div>
  </div>
</div>
`;
